### PR TITLE
Ins 12901 revert changes

### PR DESCRIPTION
**Pull request by Chrismelba-Instaclustr**

### CODE REVIEW

**Review by peter-toms — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -42,51 +44,10 @@ ifndef IC_PROV_VPC_ID
...
 install:
-	@if [ ! -d "$(HOME)/.terraform.d/plugins/" ]; then \
-		echo "$(HOME)/.terraform.d/plugins/ doesn't exist, creating..."; \
-		mkdir -p $(HOME)/.terraform.d/plugins/; \
+	@if [ ! -d $(INSTALL_FOLDER) ]; then \
+		echo "$(INSTALL_FOLDER) doesn't exist, creating..."; \
+		mkdir -p $(INSTALL_FOLDER); \
 	fi
-	cp ./bin/$(BIN_NAME)_$(VERSION) $(HOME)/.terraform.d/plugins/
+	cp ./bin/$(BIN_NAME)_v$(VERSION) $(INSTALL_FOLDER)
```
Comment (hail-insta) [inline Makefile:53]: does terraform with version <=0.12 recognise this folder?

Comment (Chrismelba-Instaclustr) [reply Makefile:53]: doubt it :P

File: acc_test/resource_cluster_test.go [hunk 2]
```diff
@@ -231,6 +234,60 @@ func TestAccClusterCustomVPC(t *testing.T) {
...
+					testCheckResourceCreated("kafka_zk", hostname, username, apiKey),
+					testCheckContactIPCorrect("kafka_zk", hostname, username, apiKey, 3),
+				),
+			},
+		},
+	})
+
+}
+
+func TestAccCassandraSparkContactPointsOdd(t *testing.T) {
```
Comment (hail-insta) [inline acc_test/resource_cluster_test.go:264]: these new tests seem to require more clusters that need to be in the RUNNING state. How much more time does that add to our acceptance test suite? we already had 4 that requires RUNNING clusters and they add significant amount of time to the testing

File: Makefile (hunk 1 above)
Comment (hail-insta) [reply Makefile:53]: Does that mean we're just "dropping" support for local terraform <=0.12 development?
I think we had an offline discussion to add a new make goal to install for versions lower than 13 for that reason.

Comment (Chrismelba-Instaclustr) [reply Makefile:53]: As discussed in standup this morning I think it's ok to drop develpment support for lower versions

File: acc_test/resource_cluster_test.go (hunk 2 above)
Comment (Chrismelba-Instaclustr) [reply acc_test/resource_cluster_test.go:264]: They do add a fair bit. I could actually move the tests to the existing clusters that need to reach running?

File: Makefile (hunk 1 above)
Comment (hail-insta) [reply Makefile:53]: I agree

File: acc_test/data/valid_with_resizable_pci_cluster.tf [hunk 3]
```diff
@@ -16,8 +16,8 @@ resource "instaclustr_cluster" "resizable_pci_cluster" {
         name = "AWS_VPC"
     }
     rack_allocation = {
-        number_of_racks = 3
-        nodes_per_rack = 1
+        number_of_racks = 4
```
Comment (hail-insta) [inline acc_test/data/valid_with_resizable_pci_cluster.tf:19]: Why was the number of racks increased?

File: acc_test/data/cassandra_contact_point.tf [hunk 4]
```diff
@@ -0,0 +1,39 @@
+provider "instaclustr" {
```
Comment (hail-insta) [inline acc_test/data/cassandra_contact_point.tf:1]: dont need this file anymore

File: acc_test/data/kafka_zk.tf [hunk 5]
```diff
@@ -0,0 +1,37 @@
+provider "instaclustr" {
```
Comment (hail-insta) [inline acc_test/data/kafka_zk.tf:1]: dont need this file anymore

File: acc_test/data/valid_with_resizable_pci_cluster.tf (hunk 3 above)
Comment (Chrismelba-Instaclustr) [reply acc_test/data/valid_with_resizable_pci_cluster.tf:19]: Just wanted to check getting IPs out with an even number (or really just any number other than 3 nodes)

**Review by arjunrajshekhar — APPROVED** [review]

**Review by hail-insta — APPROVED** [review]

**Review by arjunrajshekhar — APPROVED** [review]

**Pull request merged**